trade-finance-manager-ui/templates/case/deal/deal.njk

```diff
@@ -6,6 +6,10 @@
 
 {% extends "case/case.njk" %}
 
+{% block pageTitle -%}
+  Deal {{ deal.details.ukefDealId | dashIfEmpty }}
```

Comment:
Is the `dashIfEmpty` necessary here?

Reply:
Will this work for `deal.ukefDealId` for `GEF` deal?

Reply:
We have short time while `ukefDealId` is not available, we have dash in H1, so it should be fine to have it in head title.

Yes, it works for GEF and other deals.